FFXIVClientStructs/FFXIV/Client/Game/Event/EventHandlerModule.cs

```diff
@@ -4,8 +4,15 @@ namespace FFXIVClientStructs.FFXIV.Client.Game.Event;
 //   Client::Game::Event::ModuleBase
 // ctor "E8 ?? ?? ?? ?? 48 8B 15 ?? ?? ?? ?? 49 8D 8D ?? ?? ?? ?? 48 81 C2 ?? ?? ?? ?? E8 ?? ?? ?? ?? 48 8B 15"
 [StructLayout(LayoutKind.Explicit, Size = 0xC0)]
-public struct EventHandlerModule {
+public unsafe struct EventHandlerModule {
     [FieldOffset(0x00)] public ModuleBase ModuleBase;
     [FieldOffset(0x40)] public StdMap<uint, Pointer<EventHandler>> EventHandlerMap;
     [FieldOffset(0x50)] public StdMap<ushort, StdPair<nint, nint>> EventHandlerFactories;
+    [FieldOffset(0x60)] public CraftEventHandler* CraftEventHandler;
+    [FieldOffset(0x68)] public EventHandler* EventHandlerE0000;
+    [FieldOffset(0x70)] public EventHandler* EventHandler150001;
+    [FieldOffset(0x78)] public EventHandler* EventHandler140001;
+    [FieldOffset(0x80)] public EventHandler* EventHandler230001;
+    [FieldOffset(0x88)] public EventHandler* EventHandlerB0129;
+    [FieldOffset(0x90)] public EventHandler* EventHandlerB0130;
```

Comment:
could you name these a bit nicer, they should be (in this order):
`CraftEventHandler`
`CraftLeveEventHandler` (CraftLeveClient)
`FishingEventHandler`
`ExitRangeEventHandler`
`TripleTriadEventHandler`
`GroupPoseEventHandler` (its a CustomTalkEventHandler but its name is CmnDefGroupPose)
`IdleCameraEventHandler` (its also CustomTalkEventHandler but its name is CmnDefIdleCamera)